root/search/components/AreaResults.js

```diff
@@ -27,8 +27,7 @@ function buildResult(result, index) {
   const score = result.score;
 
   return (
-    <tr className={loopParity(index)} key={area.id}>
-      <td>{score}</td>
+    <tr className={loopParity(index)} dataScore={score} key={area.id}>
```

Comment:
I think the data attributes aren't camelCased in React for some reason, so you have to use `data-score` still.

Reply:
Fixed in https://github.com/metabrainz/musicbrainz-server/pull/695/commits/da66c002d6d50d9abb77fc4feefd5dc5efade0f9